srcs/parser/check_map.c

```diff
@@ -19,7 +19,9 @@ bool	check_map(char *file_name, t_file_data *file_data)
 		return (error_msg("Opening scene file"));
 	file_data->line = read_file(file_data);
 	if (!file_data->line)
-		return (error_msg("Reading scene file"));
+		return (!error_msg("Reading scene file"));
+	if (!read_scene_file(file_data))
+		return (!error_msg("Parsing scene file data"));
```

Comment:
do you still have the old error_msg? you should now only return (error_msg) and not (!error_msg)